### PR TITLE
refactor runtimes commands

### PR DESCRIPTION
**Pull request by itayd**
- pull test out of run. in the future, will turn this into a real unit test a-la system (ENG-545). but for now it suffices.
- also pulled the actual building into common, will be used for sessions.

### CODE REVIEW

File: cmd/ak/cmd/runtimes/test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	"github.com/spf13/cobra"
+	"golang.org/x/tools/txtar"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkbuildfile"
+)
+
+var testCmd = common.StandardCommand(&cobra.Command{
+	Use:     "test <txtar-path> [-timeout t] [--quiet]",
```
Comment (daabr) [inline cmd/ak/cmd/runtimes/test.go]:
```suggestion
	Use:     "test <txtar-path> [--timeout <t>] [--quiet]",
```

File: cmd/ak/cmd/runtimes/test.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	Short:   `Test a program`,
```
Comment (daabr) [inline cmd/ak/cmd/runtimes/test.go]:
```suggestion
	Short:   "Test a program",
```

File: cmd/ak/cmd/runtimes/test.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+		defer f.Close()
+
+		var b *sdkbuildfile.BuildFile
+
+		var expectedPrints string
+
+		bs, err := io.ReadAll(f)
+		if err != nil {
+			return fmt.Errorf("read: %w", err)
```
Comment (daabr) [inline cmd/ak/cmd/runtimes/test.go]: No need to prefix IO errors: they are very self-descriptive

Comment (itayd) [reply cmd/ak/cmd/runtimes/test.go]: done

**Review by daabr — APPROVED** [review]

**Pull request merged**